.circleci/config.yml

```diff
@@ -700,3 +700,11 @@ workflows:
     jobs:
       - install
       - publish-nightly
+
+  nightly-manual:
+    when:
+      and:
+        - equal: [ "nightly", << pipeline.parameters.GHA_Action >> ]
+        - equal: [ "workflow_dispatch", << pipeline.parameters.GHA_Event >>]
+    jobs:
+      - install
```

Comment:
will need publish-nightly here too right?

Reply:
yeah, right now im trying to see if I can get the manual trigger button to appear in the UI and test that it fires, omitted nightly for now so I don't accidentally trigger it

Reply:
Ok apparently the manual trigger button will only appear when the action yaml exists on main. Hopefully we don't run into https://github.com/orgs/community/discussions/25219 after merging